types/uni-app/tslint.json

```diff
@@ -0,0 +1,8 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
+        "max-line-length": false,
```

Comment:
Can you remove these overrides and fix up the resulting lint?

Reply:
ok，I will adjust